sc2reader/engine/plugins/context.py

```diff
@@ -151,7 +151,7 @@ def handleSelectionEvent(self, event, replay):
 
     def handleResourceTradeEvent(self, event, replay):
         event.sender = event.player
-        event.recipient = replay.players[event.recipient_id]
+        event.recipient = replay.players[event.recipient_id-1]
```

Comment:
`recipient_id` is a `pid` (`player_id`), so a lookup based on `pid` should be performed. while `replay.players` is a list of players as they appear in game, but does NOT use `pid` as its index.
This should be replaced with `replay.player[event.recipient_id]`.

see: https://github.com/ggtracker/sc2reader/blob/abe2b2b5fa455350899f4eb1518195cf2276039d/sc2reader/resources.py#L535-L536

Reply:
Done, thanks!